web/src/app/worker/methods.ts

```diff
@@ -2,12 +2,14 @@ import { useAlertCSV } from '../services/AlertMetrics/useAlertCSV'
 import { useAlertCountCSV } from '../admin/admin-alert-counts/useAlertCountCSV'
 import { useAlertMetrics } from '../services/AlertMetrics/useAlertMetrics'
 import { useAdminAlertCounts } from '../admin/admin-alert-counts/useAdminAlertCounts'
+import { useServiceMetrics } from '../admin/admin-service-metrics/useServiceMetrics'
 
 const methods = {
   useAdminAlertCounts,
   useAlertCountCSV,
   useAlertMetrics,
   useAlertCSV,
+  useServiceMetrics,
```

Comment:
Does `useServices` also need to be added here?

```suggestion
import { userServices } from '../admin/admin-service-metrics/userServices'
import { useServiceMetrics } from '../admin/admin-service-metrics/useServiceMetrics'

const methods = {
  useAdminAlertCounts,
  useAlertCountCSV,
  useAlertMetrics,
  useAlertCSV,
  useServices,
  useServiceMetrics,
```

Reply:
`useServices` isn't a worker method, it fetches the services and then the `useServiceMetrics` worker method processes the result in the background.